AdofaiTweaks/Tweaks/PlanetOpacity/PlanetOpacityPatches.cs

```diff
@@ -15,75 +15,96 @@ internal static class PlanetOpacityPatches
         private static PlanetOpacitySettings Settings { get; set; }
 
         private static readonly MethodInfo setParticleSystemColorMethod =
-            AccessTools.Method(typeof(scrPlanet), "SetParticleSystemColor");
+            AccessTools.Method(typeof(PlanetRenderer), "SetParticleSystemColor");
 
         private static readonly MethodInfo setRingColorMethod =
-            AccessTools.Method(typeof(scrPlanet), "SetRingColor");
+            AccessTools.Method(typeof(PlanetRenderer), "SetRingColor");
 
         private static readonly FieldInfo spriteField =
-            AccessTools.Field(typeof(scrPlanet), "sprite");
+            AccessTools.Field(typeof(PlanetRenderer), "sprite");
 
         private static readonly PropertyInfo spriteColorProperty =
             AccessTools.Property(spriteField.FieldType, "color");
 
-        private static float CalculateBodyOpacity(scrPlanet planet) {
-            if (planet == scrController.instance.redPlanet) {
+
+        private static float CalculateBodyOpacity(PlanetRenderer planet)
+        {
```

Comment:
please respect `.editorconfig` formatting options if possible. the braces start at the end of the line